Infrastructure/Repositories/TaskRepository.cs

```diff
@@ -91,18 +57,16 @@ public async Task<TaskModel> CreateAsync(TaskModel task)
         {
             task.Id = Guid.NewGuid();
         }
-
-        await Task.Run(() =>
-        {
-            _tasks.Add(task);
-            project.Tasks.Add(task);
-        });
+        dbContext.Tasks.Add(task);
+        project.Tasks.Add(task);
```

Comment:
Why do we need to add a task to the project.Tasks?

Reply:
Because when we add task we automatically add it to specific project

Reply:
Yep, but it should be automatic because you use foreign keys.

Reply:
got it, fixed it